src/main/java/com/codeborne/selenide/SelenideConfig.java

```diff
@@ -17,7 +17,7 @@ public class SelenideConfig implements Config {
   private String browserSize = getProperty("selenide.browserSize", "1366x768");
   private String browserVersion = getProperty("selenide.browserVersion", null);
   private String browserPosition = getProperty("selenide.browserPosition", null);
-  private boolean driverManagerEnabled = Boolean.parseBoolean(getProperty("selenide.driverManagerEnabled", "true"));
+  private boolean driverManagerEnabled = Boolean.parseBoolean(getProperty("selenide.driverManagerEnabled", "false"));
```

Comment:
`false` by default, this line can be simply deleted.

Reply:
Yes, I've deleted it.